src/app/meditation/meditation.component.ts

```diff
@@ -384,6 +433,17 @@ export class MeditationComponent {
         },
         err => console.error(err)
       );
+
+    // get commitment status for user
+    this.commitmentService.getCurrentUser()
+      .map(res => res.json())
+      .subscribe(
```

Comment:
I've added it with https://github.com/Sirimangalo/meditation-plus-angular/pull/325/commits/26294e1d36121bee880e9c783582136cdebd72df